Convex_hull_d/include/CGAL/Regular_complex_d.h

```diff
@@ -524,9 +524,7 @@ Vertex_handle  new_vertex(const Point_d& p)
         has |p| as the associated point, but is has no associated
         simplex nor index yet.}*/
 { 
-  Vertex v(p);
-  Vertex_handle h = vertices_.insert(v);
-  return h;
+  return vertices_.emplace(p);;
```

Comment:
I think you could do the same for the other overload of `new_vertex` right above, while you are there.